xfsprogs: 6.6.0 -> 6.8.0

ajs124 opened the pull request:
## Description of changes
https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/tree/doc/CHANGES?id=09ba6420a1ee2ca4bfc763e498b4ee6be415b131#n1

## Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).
---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ajs124 committed:
xfsprogs: 6.6.0 -> 6.7.0

https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/tree/doc/CHANGES?id=09ba6420a1ee2ca4bfc763e498b4ee6be415b131#n1

kirillrdy commented:
Result of `nixpkgs-review pr 304745` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>mailnagWithPlugins</li>
    <li>mailnagWithPlugins.dist</li>
    <li>rabbitvcs</li>
    <li>rabbitvcs.cli</li>
    <li>rabbitvcs.dist</li>
    <li>redpanda-server</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>19 packages failed to build:</summary>
  <ul>
    <li>ceph</li>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li>ceph.dev</li>
    <li>ceph.doc</li>
    <li>libceph (ceph.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li>ceph.man</li>
    <li>eos-installer</li>
    <li>python311Packages.jupyter-repo2docker</li>
    <li>python311Packages.jupyter-repo2docker.dist</li>
    <li>python312Packages.jupyter-repo2docker</li>
    <li>python312Packages.jupyter-repo2docker.dist</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>samba4Full</li>
    <li>samba4Full.dev</li>
    <li>samba4Full.man</li>
    <li>valent</li>
  </ul>
</details>
<details>
  <summary>201 packages built:</summary>
  <ul>
    <li>CuboCore.coreaction</li>
    <li>CuboCore.corearchiver</li>
    <li>CuboCore.corefm</li>
    <li>CuboCore.coregarage</li>
    <li>CuboCore.corehunt</li>
    <li>CuboCore.coreimage</li>
    <li>CuboCore.coreinfo</li>
    <li>CuboCore.corekeyboard</li>
    <li>CuboCore.corepad</li>
    <li>CuboCore.corepaint</li>
    <li>CuboCore.corepdf</li>
    <li>CuboCore.corepins</li>
    <li>CuboCore.corerenamer</li>
    <li>CuboCore.coreshot</li>
    <li>CuboCore.corestats</li>
    <li>CuboCore.corestuff</li>
    <li>CuboCore.coreterminal</li>
    <li>CuboCore.coretime</li>
    <li>CuboCore.coretoppings</li>
    <li>CuboCore.coreuniverse</li>
    <li>CuboCore.libcsys</li>
    <li>adapta-gtk-theme</li>
    <li>airlift</li>
    <li>airlift.dist</li>
    <li>almanah</li>
    <li>apptainer</li>
    <li>apptainer-overriden-nixos</li>
    <li>ayatana-indicator-datetime</li>
    <li>bubblemail</li>
    <li>bubblemail.dist</li>
    <li>budgie.budgie-control-center</li>
    <li>budgie.budgie-control-center.debug</li>
    <li>calls</li>
    <li>calls.devdoc</li>
    <li>cantata</li>
    <li>charliecloud</li>
    <li>chatty</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>cinnamon.nemo</li>
    <li>cinnamon.nemo-fileroller</li>
    <li>cinnamon.nemo-python</li>
    <li>cinnamon.nemo-with-extensions</li>
    <li>cinnamon.nemo.dev</li>
    <li>clementine</li>
    <li>deepin.dde-device-formatter</li>
    <li>deepin.dde-file-manager</li>
    <li>deepin.dde-gsettings-schemas</li>
    <li>deepin.deepin-compressor</li>
    <li>deepin.deepin-music</li>
    <li>deepin.udisks2-qt5</li>
    <li>deepin.util-dfm</li>
    <li>devcontainer</li>
    <li>docker</li>
    <li>docker-gc</li>
    <li>docker-sbom</li>
    <li>docker_25</li>
    <li>docker_26</li>
    <li>endeavour</li>
    <li>enlightenment.enlightenment</li>
    <li>evolution</li>
    <li>evolution-data-server</li>
    <li>evolution-data-server-gtk4</li>
    <li>evolution-data-server-gtk4.dev</li>
    <li>evolution-data-server.dev</li>
    <li>evolution-ews</li>
    <li>evolutionWithPlugins</li>
    <li>exaile</li>
    <li>fn-cli</li>
    <li>folks</li>
    <li>folks.dev</li>
    <li>folks.devdoc</li>
    <li>gfbgraph</li>
    <li>gfbgraph.dev</li>
    <li>gfbgraph.devdoc</li>
    <li>gitg</li>
    <li>glusterfs</li>
    <li>gnome-browser-connector</li>
    <li>gnome-multi-writer</li>
    <li>gnome-online-accounts</li>
    <li>gnome-online-accounts.debug</li>
    <li>gnome-online-accounts.dev</li>
    <li>gnome-online-accounts.devdoc</li>
    <li>gnome-online-accounts.man</li>
    <li>gnome-photos</li>
    <li>gnome-photos.installedTests</li>
    <li>gnome-recipes</li>
    <li>gnome.cheese</li>
    <li>gnome.cheese.devdoc</li>
    <li>gnome.cheese.man</li>
    <li>gnome.geary</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-control-center.debug</li>
    <li>gnome.gnome-disk-utility</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-notes</li>
    <li>gnome.gnome-online-miners</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-panel-with-modules</li>
    <li>gnome.gnome-panel.dev</li>
    <li>gnome.gnome-panel.man</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-session.debug</li>
    <li>gnome.gnome-session.sessions</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-shell.debug</li>
    <li>gnome.gnome-shell.devdoc</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gvfs</li>
    <li>gnome.gvfs.debug</li>
    <li>gnome.nixos-gsettings-overrides</li>
    <li>gnome.totem</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.gsconnect.installedTests</li>
    <li>grilo-plugins</li>
    <li>gvfs</li>
    <li>gvfs.debug</li>
    <li>kdePackages.kpmcore</li>
    <li>kdePackages.kpmcore.debug</li>
    <li>kdePackages.kpmcore.dev</li>
    <li>kdePackages.partitionmanager</li>
    <li>kdePackages.partitionmanager.debug</li>
    <li>kdePackages.partitionmanager.dev</li>
    <li>libgdata</li>
    <li>libgdata.dev</li>
    <li>libgdata.installedTests</li>
    <li>libsForQt5.partitionmanager</li>
    <li>libxfs (libxfs.bin ,libxfs.dev ,libxfs.doc ,xfsprogs.dev)</li>
    <li>libzapojit</li>
    <li>libzapojit.dev</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-security-privacy</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.mediascanner2</li>
    <li>lomiri.mediascanner2.dev</li>
    <li>mate.mate-control-center</li>
    <li>mate.mate-panel-with-applets</li>
    <li>mate.mate-settings-daemon-wrapped</li>
    <li>mate.mate-utils</li>
    <li>mediawriter</li>
    <li>mojave-gtk-theme</li>
    <li>monitor</li>
    <li>nfs-ganesha</li>
    <li>nfs-ganesha.tools</li>
    <li>ns-usbloader</li>
    <li>nvidia-docker</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-tasks</li>
    <li>pantheon.switchboard-plug-about</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>phosh</li>
    <li>phosh-mobile-settings</li>
    <li>pipework</li>
    <li>planify</li>
    <li>plasticity</li>
    <li>psensor</li>
    <li>rapid-photo-downloader</li>
    <li>rapid-photo-downloader.dist</li>
    <li>singularity</li>
    <li>singularity-overriden-nixos</li>
    <li>spaceFM</li>
    <li>stratisd</li>
    <li>stratisd.initrd</li>
    <li>udiskie</li>
    <li>udiskie.dist</li>
    <li>udisks</li>
    <li>udisks.dev</li>
    <li>udisks.devdoc</li>
    <li>udisks.man</li>
    <li>udocker</li>
    <li>udocker.dist</li>
    <li>usbimager</li>
    <li>usermount</li>
    <li>ventoy-full</li>
    <li>vifm-full</li>
    <li>vimix-gtk-themes</li>
    <li>whitesur-gtk-theme</li>
    <li>wipefreespace</li>
    <li>xfsdump</li>
    <li>xfsprogs</li>
    <li>xfsprogs.bin</li>
    <li>xfsprogs.doc</li>
    <li>xfstests</li>
  </ul>
</details>

Lassulus commented:
it seems that this PR broke Ceph

ajs124 committed:
xfsprogs: 6.7.0 -> 6.8.0

https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/tree/doc/CHANGES?id=df4bd2d27189a98711fd35965c18bee25a25a9ea

Lassulus commented:
hmm, nix-review wasn't able to build cepth, but building manually ceph in the nixpkgs checkout nixpkgs-review creates worked. So I guess it would work and the problem with nixpkgs-review is the that too many builds are started at the same time or something like that.
